neurom/io/tests/test_h5_reader.py

```diff
@@ -40,6 +40,7 @@
 H5_PATH = os.path.join(DATA_PATH, 'h5')
 H5V1_PATH = os.path.join(H5_PATH, 'v1')
 H5V2_PATH = os.path.join(H5_PATH, 'v2')
+SWC_PATH = os.path.join(DATA_PATH, 'swc')
 
 
 def test_read_h5v1_basic():
```

Comment:
I think these test_h5v* are missing the no duplicate removal tests. Other than that it looks fine!